Mathlib/Data/Polynomial/RingDivision.lean

```diff
@@ -355,32 +364,36 @@ section CommRing
 
 variable [CommRing R]
 
+/- Porting note: due to a diamond, introduced 
```

Comment:
I don't understand how this porting note relates to neighbouring lines or the diff with mathport-output.